source/documentation/using_the_api/item_entry_record_example.md

```diff
@@ -0,0 +1,26 @@
+## The relationship between entries, items and records
+
+Registers are made up of:
+
+* entries
+* items
+* records
+
+Entries contain metadata about what and when data has changed. 
+
+Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+
+Records represent things that change over time. Records are identifiable by keys and correspond to the latest entry for a particular key, as indicated by the entry number.
+
+### A worked example
+
+The `country` register, and one of its keys, ‘CI’, provides a useful example. 
+
+Using the `GET /entries/{entry-number}` where `{entry-number}` is '90' returns the [90th entry to the `country` register](https://country.register.gov.uk/entries/90.json), which contains metadata. Using the `GET /items/{item-hash}` endpoint on that entry returns the corresponding item, which reveals that the `official-name` field for this entry is ‘The Republic of Cote D’Ivoire’. 
```

Comment:
I think some more links would help.  For example, a link to the [corresponding item](https://country.register.gov.uk/items/sha-256:7c16257bd45b4716914010b39dd40e5a6b985b8928d7b8bb0fe3005d2f2b0fec.json) of the 90th entry would clarify that `sha-256:` is part of the item hash (I just made this mistake).